### PR TITLE
fix callback issue #25

### PR DESCRIPTION
**Pull request by fbandrey**
`callback_url` method has been removed from `omniauth-oauth2`.
Authorization is broken for now, this is hotfix.
Rubocop and tests are passed.

### CODE REVIEW

File: omniauth-gplus.gemspec [hunk 1]
```diff
@@ -26,7 +26,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'kramdown'
   s.add_development_dependency 'pry'
   s.add_development_dependency 'coveralls'
-  s.add_development_dependency 'rubocop'
+  s.add_development_dependency 'rubocop', '0.24.1'
```
Comment (samdunne) [inline omniauth-gplus.gemspec]: Why are we version locking?

Comment (fbandrey) [inline omniauth-gplus.gemspec]: @samdunne In order to pass all travis builds. The next one requires ruby's version more or equal 1.9.3, and build for 1.9.2 always failed. https://rubygems.org/gems/rubocop/versions/0.25.0

Comment (samdunne) [inline omniauth-gplus.gemspec]: We should probably deprecate anything < 1.9.3 to be honest

Comment (fbandrey) [inline omniauth-gplus.gemspec]: @samdunne Yep, good idea. Will do this shortly.

Comment (fbandrey) [inline omniauth-gplus.gemspec]: Oh, sorry, config inside repo. I've removed 1.9.2.

**Comment by fbandrey** [comment]
@samdunne Any more blockers on this?

**Comment by samdunne** [comment]
Nope. Merged :)

**Pull request merged**